src/app/modules/product/_model/product.ts

```diff
@@ -0,0 +1,13 @@
+import { ProductImage } from "./product-image";
+
+export class Product {
+    product_id: number = 0;
+    product: string = "";
+    gtin: string = "";
+    description: string = "";
+    price: number = 0;
+    stock: number = 0;
+    category_id: number = 0;
+    image: ProductImage = new ProductImage();
```

Comment:
Product no tiene un atributo `image`
```suggestion
    category: string = "";
```